VagrantBuild.sh

```diff
@@ -23,7 +23,9 @@ if [ -f missing ]; then
   rm -f missing
 fi
 
-aclocal && autoconf && autoheader && automake --add-missing --copy && ./configure --quiet --with-rnd --with-services --with-uitests
+# Taking out ui-tests until we get Tomcat8 etc installed
+# aclocal && autoconf && autoheader && automake --add-missing --copy && ./configure --quiet --with-rnd --with-services --with-uitests
+aclocal && autoconf && autoheader && automake --add-missing --copy && ./configure --with-rnd --with-services --with-uitests
```

Comment:
Looks like this can be reverted.  All that is turned off is the `--quiet` flag and now that `ui-tests` have been enabled (Tomcat8 is installed) it can be changed back.